src/nn/Model.js

```diff
@@ -58,6 +59,13 @@ export default class Model {
     return ResultCode.NO_ERROR;
   }
 
+  /**
+   * Determine if the OperandCode of model is quant8.
+   */
+  isQuant8() {
+    return this._operands[this._inputs[0]].type === OperandCode.TENSOR_QUANT8_ASYMM || this._isQuantized ? true : false;
```

Comment:
Can it be simplified to 
```js
return this._isQuantized;
```

Reply:
> How about we check every operand when invoking `addOperand`.

Because I only check the operand for `add` operation according to your comment or maybe I just understand wrong. If we check every single operand for every operation the code can be simplified as you mentioned above. Do we need to do that?

Reply:
> Because I only check the operand for `add` operation according to your comment or maybe I just understand wrong.

I mean checking operand type in `addOperand` API not just for `ADD` operation. 

> If we check every single operand for every operation the code can be simplified as you mentioned above. Do we need to do that?

Yes. Please do that.